pkg/backends/jmeter/resources.go

```diff
@@ -416,29 +406,6 @@ func splitTestData(testdata string, n int, logger *zap.Logger) ([][][]string, er
 	return chunks, nil
 }
 
-// ReadSecret reads data from csv file to save it as a map for creating a secret
-func readSecret(envVars string, logger *zap.Logger) (map[string]string, error) {
-	m := make(map[string]string)
-
-	reader := csv.NewReader(strings.NewReader(envVars))
-	for {
-		line, err := reader.Read()
-		if err == io.EOF {
-			break
-		}
-		if err != nil {
-			logger.Error("Error on reading envVars csv file ", zap.Error(err))
-			return nil, err
-		}
-		if len(line) != 2 {
-			logger.Error("Invalid secret format in envVars file")
-			return nil, os.ErrInvalid
-		}
-		m[line[0]] = line[1]
-	}
-	return m, nil
-}
-
```

Comment:
Move this function to a dedicated package called `helper`, so Locust can make use of it too.